doc/sphinx/addendum/generalized-rewriting.rst

```diff
@@ -582,6 +584,8 @@ Deprecated syntax and backward incompatibilities
 .. cmd:: Add Morphism @ident : @ident
    :name: Add Morphism
 
+  .. deprecated:: 8.8
+
```

Comment:
Following #8713, it might be time to decide to actually undeprecate these two commands.

Reply:
Ok, but let's discuss it in a separate PR.